subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/KotlinInitScript.kt

```diff
@@ -55,6 +55,7 @@ import kotlin.script.templates.ScriptTemplateDefinition
 /**
  * Script template for Kotlin init scripts.
  */
+@Suppress("deprecation")
```

Comment:
:x: this is a bit too broad and can be put on the deprecated type usage site instead

Line 71 below:
```kotlin
) : @Suppress("deprecation") InitScriptApi(host.target) {
```

same in other script templates

Reply:
Thanks.

Reply:
Done.